### ISSUE TITLE
 Undefined method emit_forward_arg= in AST::Builder:Class

### ISSUE THREAD

**Issue by wintersolutions** [rubocop-hq/rubocop-ast#164]
I'm getting the error message:
```
undefined method `emit_forward_arg=' for RuboCop::AST::Builder:Class (NoMethodError)
```
with the `parser` v2.7.0.2 gem. If I update to `parser` v3 the error goes away.

* ruby 2.6.6
* rubocop 1.7.0
* rubocop-ast 1.3.0

Full trace:
```
Traceback (most recent call last):
	15: from /home/my_home/.rvm/gems/ruby-2.6.6/bin/ruby_executable_hooks:24:in `<main>'
	14: from /home/my_home/.rvm/gems/ruby-2.6.6/bin/ruby_executable_hooks:24:in `eval'
	13: from /home/my_home/.rvm/gems/ruby-2.6.6/bin/rubocop:23:in `<main>'
	12: from /home/my_home/.rvm/gems/ruby-2.6.6/bin/rubocop:23:in `load'
	11: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-1.7.0/exe/rubocop:6:in `<top (required)>'
	10: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-1.7.0/exe/rubocop:6:in `require'
	 9: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-1.7.0/lib/rubocop.rb:14:in `<top (required)>'
	 8: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-1.7.0/lib/rubocop.rb:14:in `require'
	 7: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop-ast.rb:3:in `<top (required)>'
	 6: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop-ast.rb:3:in `require_relative'
	 5: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop/ast.rb:81:in `<top (required)>'
	 4: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop/ast.rb:81:in `require_relative'
	 3: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop/ast/builder.rb:3:in `<top (required)>'
	 2: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop/ast/builder.rb:4:in `<module:RuboCop>'
	 1: from /home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop/ast/builder.rb:16:in `<module:AST>'
/home/my_home/.rvm/gems/ruby-2.6.6/gems/rubocop-ast-1.3.0/lib/rubocop/ast/builder.rb:17:in `<class:Builder>': undefined method `emit_forward_arg=' for RuboCop::AST::Builder:Class (NoMethodError)
```

**Comment by marcandre** [comment]
Thanks for the report.

I recommend always upgrading `parser` as it is always compatible with previous versions. That being said, it's an easy fix to avoid this error, so I'll do that with `rubocop-ast`.

**Comment by marcandre** [comment]
Released 1.4.0

**Comment by wintersolutions** [comment]
Thank you very much!
